### PR TITLE
Feature two fluid darcy losses

### PR DESCRIPTION
**Pull request by jcotela**
Adding a new term to model porous losses in the two fluid element.
Opening the Pull Request before the code becomes too detached from master.

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_elements/two_fluid_vms.h [hunk 1]
```diff
@@ -1,44 +1,18 @@
...
+//                   Multi-Physics  
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: jcotela $
-//   Date:                $Date: 2010-10-09 10:34:00 $
-//   Revision:            $Revision: 0.1 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/custom_elements/two_fluid_vms.h:10]: I suggest

Jordi Cotela
Riccardo Rossi

File: applications/FluidDynamicsApplication/tests/TwoFluidDarcyTest/reference_darcy_density.csv [hunk 2]
```diff
@@ -0,0 +1,447 @@
+#ID, PRESSURE
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/tests/TwoFluidDarcyTest/reference_darcy_density.csv]: just as a suggestion, wouldn't it make sense to use the check process that Vicente did? it has the mechanism to verify this

File: applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py [hunk 3]
```diff
@@ -49,6 +50,7 @@ def AssambleTestSuites():
     smallSuite.addTest(TNavierStokesWallConditionTest('test_execution'))
     smallSuite.addTest(BuoyancyTest('testEulerian'))
     smallSuite.addTest(BuoyancyTest('testThermalExpansionCoefficient'))
+    smallSuite.addTest(DarcyChannelTest('testDarcyDensity'))
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py:53]: are u sure it should be in the "smallSuite"? it is not really small i think

**Review by RiccardoRossi — APPROVED** [review]
some minor comments, mostly cosmetic

File: applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py (hunk 3 above)
Comment (jcotela) [reply applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py:53]: Well no, I'm not sure. It is smaller than several of the existing "small" tests, though...
I'd say we leave this discussion for now and move it where appropriate within #872.

File: applications/FluidDynamicsApplication/tests/TwoFluidDarcyTest/reference_darcy_density.csv (hunk 2 above)
Comment (jcotela) [reply applications/FluidDynamicsApplication/tests/TwoFluidDarcyTest/reference_darcy_density.csv]: We'd still need to store the reference values somehow...
Something that might be smarter is to compare the pressure drop to theoretical (which could be done with just two points) avoiding the reference files altogether.

File: applications/FluidDynamicsApplication/custom_elements/two_fluid_vms.h (hunk 1 above)
Comment (jcotela) [reply applications/FluidDynamicsApplication/custom_elements/two_fluid_vms.h:10]: I still consider the two-fluid element "your child" :P

**Pull request merged**